internal/kubernetes/nodefilters.go

```diff
@@ -60,6 +67,25 @@ func NewNodeConditionFilter(ct []string) func(o interface{}) bool {
 	}
 }
 
+// ParseConditions can parse the string array of conditions to
+func ParseConditions(conditions []string) []SuppliedCondition {
+	parsed := make([]SuppliedCondition, len(conditions))
+	for i, c := range conditions {
+		ts := strings.SplitN(c, "=", 2)
+		if len(ts) != 2 {
+			// Keep backward compatibility
+			ts = []string{c, "True,0"}
+		}
+		sm := strings.SplitN(ts[1], ",", 2)
+		m, err := strconv.ParseInt(sm[1], 10, 64)
```

Comment:
golang has a fairly pervasive pattern of supporting duration strings such as `10s` `1m` etc. I'd prefer to use `time.ParseDuration` here. I tend to believe it makes the command line parameters more obvious e.g. you'll pass the conditions as `OutofDisk=True,10s` which makes it obvious I'm dealing with seconds instead of whatever interval we decided it represents.

In turn, you could change the type of `MinimumWaitingTime` to `time.Duration` and remove the `time.Duration` cast on [line 60 in nodefilters.go](https://github.com/planetlabs/draino/pull/43/files#diff-b5f581068d88d5e78a05c62adbd0a9aaR60).